mlir/include/mlir/Conversion/IndexToSPIRV/IndexToSPIRV.h

```diff
@@ -0,0 +1,28 @@
+//===- IndexToSPIRV.h - Index to SPIRV dialect conversion -------*- C++ -*-===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef MLIR_CONVERSION_INDEXTOSPIRV_INDEXTOSPIRV_H
+#define MLIR_CONVERSION_INDEXTOSPIRV_INDEXTOSPIRV_H
+
+#include <memory>
```

Comment:
IIRC, I added this because it is required when we include `Passes.h.inc` below. I have updated to include the declaration that requires it to this header as well to make it clear.